Tests fail with geopandas 0.11

It appears that an expected warning is no longer raised:
```
_______________________ TestClip.test_clip_init_default ________________________

self = <mixin_tests.TestClip testMethod=test_clip_init_default>

    def test_clip_init_default(self):
        clipmixin = self.create_clipmixin()
    
        # UserWarning because we have a narrow clip
>       with pytest.warns(UserWarning):
E       Failed: DID NOT WARN. No warnings of type (<class 'UserWarning'>,) were emitted. The list of emitted warnings is: [].

tests/mixin_tests.py:593: Failed
```

--- a/tests/mixin_tests.py
+++ b/tests/mixin_tests.py
@@ -589,9 +589,8 @@ class TestClip(unittest.TestCase):
     def test_clip_init_default(self):
         clipmixin = self.create_clipmixin()
 
-        # UserWarning because we have a narrow clip
-        with pytest.warns(UserWarning):
-            df_result = clipmixin.set_clip(clipmixin.df)
+        # UserWarning because we have a narrow clip (before geopandas 0.11)
+        df_result = clipmixin.set_clip(clipmixin.df)
         expected = Polygon([[0, 0], [0, 100], [100, 100], [100, 0]])
         result = df_result.geometry.unary_union.envelope
         assert expected.contains(result)

